app/views/layouts/_toast.html.erb

```diff
@@ -1,31 +1,25 @@
+
 <% if flash.any? %>
-  <div class="fixed top-4 left-[50%] translate-x-[-50%] w-full sm:w-1/2
-              flex flex-col gap-4 items-center
-              z-10
-              ">
-      <% flash.each do |type, text| %>
-        <%
-          toast_classes = case type
-                         when "notice"
-                           "bg-blue-100 text-blue-900 border-blue-400"
-                         when "alert"
-                           "bg-red-100 text-red-900 border-red-400"
-                         end
-        %>
-      <div
-          class="<%= toast_classes %> flex justify-between gap-4 rounded border items-baseline w-full pl-4"
-          data-controller="click-toggle"
-          data-click-toggle-flippable-class="hidden"
-          data-click-toggle-target="destination"
-      >
-          <p><%= text %></p>
-          <%= button_tag type: "button",
-                data: { action: "click-toggle#toggleClass" },
-                class: "p-4" do
-          %>
-            &times;
-          <% end %>
-        </div>
-      <% end %>
+  <div class="toast toast-top toast-end transition-transform duration-100 ease-in"
+        data-controller="transition"
+        data-transition-toggle-class="translate-x-full"
+        data-transition-target="me"
+        data-transition-after-timeout-value="3000"
+  >
+    <% flash.each do |type, text| %>
+      <%
+        alert_class = case type
+        when "notice"
+          "alert-info"
+        when "alert"
+          "alert-error"
+        else
+          "alert-info"
+        end
+      %>
+      <div class="alert <%= alert_class %>">
+        <span><%= text %></span>
+      </div>
```

Comment:
This toast component comes from the DaisyUI library. It slides in from the top-right of the screen and slides back out when it auto-dismisses.